### DISCUSSION TITLE
Edit Project Root

### DISCUSSION THREAD

**Discussion by phistep** [zed-industries/zed#21518] [Feature Requests]
I added a couple of folders to my project, but I realized later, that I started with the wrong folder. Now the terminal always opens in the "wrong" dir, the topleft workspace name is the "wrong" one, and the project-level git branch also belongs to the "wrong" folder. How can I configure which of the folders in the project is the project root?

![image](https://github.com/user-attachments/assets/c3f3b0fd-124c-4d83-97d9-64896fd26fda)

I've been poking around `~/.local/share/zed/0-stable/db.sqlite`'s `workspaces` folder, but the `local_paths` column is a binary blob in an (to me) unknown format, so it's not straight forward to edit.

![image](https://github.com/user-attachments/assets/0a68ac85-3f33-4271-aae6-f6b682167bcf)

Currently `/home/philippstephan/src/libshumate` is the project root, but I would like to change it to `/home/philippstephan/src/map-overlay`.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
